extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-attachment.js

```diff
@@ -275,5 +277,18 @@ describes.realWin(
         StoryAnalyticsEvent.SHOPPING_PDP_VIEW
       );
     });
+
+    it('should call history service on Product Listing Page card click', async () => {
+      const historyStub = env.sandbox.stub(history, 'setHistoryState');
+
+      await layoutShoppingImplAndAttachmentChildImpl();
+      storeService.dispatch(Action.TOGGLE_PAGE_ATTACHMENT_STATE, true);
+      const plpCard = attachmentChildEl.querySelector(
+        '.i-amphtml-amp-story-shopping-plp-card'
+      );
+      plpCard.dispatchEvent(new Event('click'));
+
+      expect(historyStub).to.have.been.called;
```

Comment:
Would it be worth using `calledWith(...)` to check which arguments were passed?

Reply:
Yes, added the checks.